dev-util/bcc/bcc-0.24.0-r1.ebuild

```diff
@@ -94,7 +94,7 @@ src_configure() {
 	local bcc_python_impls
 	python_foreach_impl python_add_impl
 
-	local -a mycmakeargs=(
+	local mycmakeargs=(
```

Comment:
I suggest cleanup old instead but all lgtm anyway.

Reply:
removed the old one